server/utils/firebase_algolia_server_code/functions/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Description: Turn ad entities into algolia records
+ *              With HTTP cloud functions  
+ *              Basically compile with `firebase deploy --only functions`
+ *              These will then appear in functions tab of firestore 
+ * Author: Robert Marcus
+ * Date: June 26, 2020
+ * Note: Current algolia index is dev_ADS
+ */
+
+const algoliasearch = require('algoliasearch');
+const algoliaFunctions = require('./algoliaFunctions');
+const functions = require('firebase-functions');
+
+/**
+ * Looking to compile this on your local machine but don't have the keys?
```

Comment:
Maybe a little informal? I'm not completely sure about the tone we're supposed to take with these kinds of comments.

Reply:
Rewrote this a little to be less informal.